terraform/table_terraform_output.go

```diff
@@ -145,15 +177,25 @@ func buildOutput(ctx context.Context, path string, content []byte, name string,
 	// Remove all "_kics" arguments
 	sanitizeDocument(d)
 
-	start, end, source, err := getBlock(ctx, path, content, "output", []string{name})
-	if err != nil {
-		plugin.Logger(ctx).Error("terraform_output.buildOutput", "getBlock", err)
-		return tfOutput, err
+	if isTFStateFilePath {
+		file, err := os.Open(path)
+		if err != nil {
+			plugin.Logger(ctx).Error("terraform_output.listOutputs", "open_file_error", err, "path", path)
+			return tfOutput, err
+		}
+		startLine, endLine := findBlockLinesFromJSON(file, "outputs", name)
+		tfOutput.StartLine = startLine
+		tfOutput.EndLine = endLine
```

Comment:
Are we able to get the `Source` info too?

Reply:
Added source info.